src/concepts/tfgrid_by_design.md

```diff
@@ -0,0 +1,147 @@
+
+To build a reliable deployment on the TFGrid, you need to take into consideration the three different types of unit on the TFGrid: storage, compute and network. Let's delve into these a little bit more.
+
+# TFGrid Main Components Overview and Examples
+
+We provide here an overview of some of the main components of the TFGrid. We also provide examples for each of those components in order for the reader to obtain a clear understanding of the TF Ecosystem. By understanding the different components of the TFGrid, system administrators will be able to deploy resilient, redundant and reliable solutions on the grid. 
+
+For a complete list of the TFGrid components, read [this documentation](./grid3_components.md).
+
+## Storage Units
+
+Storage units are related to the data stored in SSD and HDD disks. The Quantum Safe Filesystem (QSFS) technology developed by ThreeFold ensures redundancy and resilience in storage units. If one disk of the QSFS array goes offline, the rest of the system can still function properly. To the contrary, if a user stores information on one single 3Node and this 3Node has a drastic disk failure, the user will lose the data. Another way to achieve redundancy in the storage category would be to deploy a solution with real-time synced databases of two or more 3nodes connected via a wireguard VPN. Note that other configurations offering reliability and redundancy are possible.
+
+Let's explore some storage components of the ThreeFold Grid.
+
+### 0-DB-FS
+
+[0-DB-FS](./grid3_components.md#0-db-fs) is storage system that allows for efficient and secure storage of files on the ThreeFold Grid. 0-DB-FS is built on top of 0-DB, which is a key-value store optimized for high performance and scalability. It provides a decentralized and distributed approach to file storage, ensuring data redundancy and availability.
+
+### 0-stor_v2
+
+[0-stor_v2](./grid3_components.md#0-stor_v2) is a distributed and decentralized storage solution that enables data storage and retrieval on the ThreeFold Grid. 
+
+### QSFS
+
+[Quantum Safe Filesystem (QSFS)](./grid3_components.md#qsfs) is ThreeFold's innovative storage solution designed to address the security challenges posed by quantum computing. QSFS employs advanced cryptographic techniques that are resistant to attacks from quantum computers, ensuring the confidentiality and integrity of stored data. By its design, QSFS also offers a high level of redundancy.
+
+## Compute Units
+
```

Comment:
Could you please include some information especially about VCPU?

Reply:
I found some data about VCPUs, just add brief about it. 
 
Virtual CPUs (vCPUs), They are virtual representations of physical CPUs that allow multiple virtual machines (VMs) to run concurrently on a single physical server or host.
Virtualization platforms, allocate vCPUs to each VM, enabling them to execute tasks and run applications as if they were running on dedicated physical hardware. The number of vCPUs assigned to a VM determines its processing power and capacity to handle workloads.

And also if possible mention max number of VCPUs a node can have, More info https://github.com/threefoldtech/zos/issues/2056

Reply:
Thanks for the proposition and the text. I pushed a new commit with this. Let me know if it's OK!